app/views/admin/base/_tree_item.html.haml

```diff
@@ -0,0 +1,10 @@
+%li.sort-tree__item(data-sort-tree-item){id: "list_#{item.id}"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))